tests/e2e/v4/state.go

```diff
@@ -80,20 +80,6 @@ func (tr Commands) GetBlockHeight(chain ChainID) uint {
 	return uint(blockHeight)
 }
 
-func (tr Commands) waitUntilBlock(chain ChainID, block uint, timeout time.Duration) {
```

Comment:
Can we remove those as is? Are they needed for any compatibility tests?

Reply:
Nothing calls this method.